pkgs/top-level/all-packages.nix

```diff
@@ -11643,6 +11643,8 @@ with pkgs;
 
   ssh-copy-id = callPackage ../tools/networking/openssh/copyid.nix { };
 
+  sshd-openpgp-auth = callPackage ../by-name/ss/ssh-openpgp-auth/daemon.nix { };
```

Comment:
instead of using `all-packages.nix` (which can cause issues like merge conflicts), how about moving `daemon.nix` to `by-name/ss/sshd-openpgp-auth/package.nix`?

Reply:
Discussed in the other thread.